tests/driver_kw2xrf/Makefile

```diff
@@ -1,7 +1,7 @@
 include ../Makefile.tests_common
 
 USEMODULE += auto_init_gnrc_netif
-USEMODULE += gnrc_netdev_default
+USEMODULE += gnrc
```

Comment:
Mhhh why was this test pulling in `gnrc_netdev_default` anyway? Shouldn't it rather pull in `kw2rxf`?

Reply:
it does a few lines further down.
I think this part was originally just copy & paste from some example

Reply:
Ah, I was confused because it was assigning to `DRIVER` first (probably copy-pasted from the `at86rf2xx` test) and then only later to `USEMODULE`